src/index.html

```diff
@@ -252,65 +252,64 @@ <h4 class="padding-bottom">Structural variants studies (&gt;50bp)</h4>
+                                <li>Prepare valid VCF file(s), which can be validated prior to submission using the <a href="https://github.com/EBIvariation/eva-sub-cli" target="_blank">eva-sub-cli tool</a>.</li>
+                                <li>Complete a <a href="https://raw.githubusercontent.com/EBIvariation/eva-sub-cli/main/eva_sub_cli/etc/EVA_Submission_template.xlsx" target="_blank">metadata template</a> describing the samples and analyses in your study. This can be validated alongside your VCF files with <a href="https://github.com/EBIvariation/eva-sub-cli" target="_blank">eva-sub-cli</a>. Please provide as much metadata as possible since this information is useful for downstream analysis and leads to increased dataset reuse. For reference, here is an <a href="https://www.ebi.ac.uk/eva/files/EVA_Submission_template.V1.1.5_mockup.xlsx" target="_blank"><b>example</b></a> of a completed metadata template.</li>
+                                <li>Please also note that the template requires the submitter to fill in some personal data, which will be used as described in our <a href="https://www.ebi.ac.uk/data-protection/privacy-notice/eva-submission-service" target="_blank">privacy notice</a>.</li>
+                            </ul>
+                        </div>
+                        <div class="arrow"></div>
+                        <div class="row">
+                            <div class="box blue">
+                                <h3>💻 Submit via eva-sub-cli</h3>
+                                <p>This is our command-line tool for validating and uploading your data and metadata. For instructions on how to install and use <a href="https://github.com/EBIvariation/eva-sub-cli" target="_blank">eva-sub-cli</a>, please see our <a href="https://github.com/EBIvariation/eva-sub-cli/blob/main/README.md" target="_blank">documentation</a>.</p>
+                                <p>This method has faster turnaround times, but may be more suitable for more technical users.</p>
+                                <p>Before submitting data through eva-sub-cli, please make sure you have a <a href="https://www.ebi.ac.uk/ena/submit/webin/login" target="_blank">Webin</a> Account.</p>
+                            </div>
+                            <div class="or-label">OR</div>
+                            <div class="box blue">
+                                <h3>✉️ Submit via email</h3>
+                                <p>Contact <a href="mailto:eva-helpdesk@ebi.ac.uk" target="_top">eva-helpdesk@ebi.ac.uk</a> to request a submission. You will receive a custom private FTP account to deposit your data and metadata files, as well as further instructions.</p>
+                                <p>This submission method is less technical but may involve longer processing times as we help you validate your data.</p>
+                                <p>Please make sure to read the instructions given in the metadata spreadsheet before submitting.</p>

```

Comment:
I feel like the last point should go in the Prepare box, since it really applies to both paths

Reply:
As discussed in the standup, leaving as it is for now